packages/messenger-widget/src/contexts/DM3Configuration.ts

```diff
@@ -0,0 +1,39 @@
+import { log } from '@dm3-org/dm3-lib-shared';
+import {
+    DM3ConfigurationStateActions,
+    DM3ConfigurationStateType,
+} from '../utils/enum-type-utils';
+import { DM3Configuration } from '../interfaces/config';
+
+export function dm3ConfigurationReducer(
```

Comment:
Create a hook similar to useConversations or useMessage instead of using the legacy state

Reply:
oh yes, that makes sense